src/high-level/vector.lisp

```diff
@@ -12,6 +12,9 @@
                    (:copier nil))
   (size 0 :type alexandria:positive-fixnum))
 
+(defmethod layout ((v vector))
+  :row-major)
```

Comment:
Is there any reason for this particular choice? (I hope not...)

Reply:
Of course not.